src/redux/books/book.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable arrow-body-style */
+const InitialState = [];
+
+export const addBook = (id, author, title) => {
+  return {
+    type: 'ADDED_BOOK',
+    payload: { id, author, title },
+  };
+};
+
+export const removeBook = (id, author, title) => {
+  return {
+    type: 'REMOVED_BOOK',
+    payload: { id, author, title },
+  };
+};
```

Comment:
- [ ] Kindly make sure that the type follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux).